format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSourceConfig.java

```diff
@@ -94,6 +94,12 @@
     + "is anything other than 'delimited'.")
   private String delimiter;
 
+  @Macro
+  @Nullable
+  @Description("Whether to split the content between quotes. This value will only be used if the format is 'csv', " +
+                 "'tsv' or 'delimited'. Default value is true.")
```

Comment:
Shouldn't the default be false?

Reply:
that will change the current behavior since without this change, we will split the quotes